components/status-indicator/test/status-indicator.html

```diff
@@ -0,0 +1,100 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+	<title>d2l-status-indicator unit tests</title>
+	<script src="/node_modules/@webstatusIndicators/webstatusIndicatorsjs/webstatusIndicators-loader.js"></script>
+	<script src="/node_modules/mocha/mocha.js"></script>
+	<script src="/node_modules/chai/chai.js"></script>
+	<script src="/node_modules/@polymer/test-fixture/test-fixture.js"></script>
+	<script src="/node_modules/wct-mocha/wct-mocha.js"></script>
+	<script type="module" src="../status-indicator.js"></script>
+</head>
+
+<body>
+
+	<test-fixture id="subtle">
+		<template>
+			<d2l-status-indicator state="default" text="test subtle"></d2l-status-indicator>
+		</template>
+	</test-fixture>
+
+	<test-fixture id="bold">
+		<template>
+			<d2l-status-indicator state="default" text="test bold" bold></d2l-status-indicator>
+		</template>
+	</test-fixture>
+
+	<script type="module">
+		import { runAxe } from '../../../tools/a11y-test-helper.js';
+
+		describe('d2l-status-indicator', () => {
+
+			let statusIndicator;
+
+			describe('subtle', () => {
+
+				beforeEach(async() => {
+					statusIndicator = fixture('subtle');
+					await statusIndicator.updateComplete;
+				});
+
+				it('passes all axe tests', async() => {
+					await runAxe(statusIndicator);
+				});
+
+				it('should set the demo text', () => {
+					expect(statusIndicator.getAttribute('text')).to.equal('test subtle');
+				});
+
+				it('should have the initially specified default state', () => {
+					expect(statusIndicator.getAttribute('state')).to.equal('default');
+				});
+
+				it('should have state "success" when specified', async() => {
+					statusIndicator.state = 'success';
+					await statusIndicator.updateComplete;
+					expect(statusIndicator.getAttribute('state')).to.equal('success');
+				});
+
+				it('should have state "alert" when specified', async() => {
+					statusIndicator.state = 'alert';
+					await statusIndicator.updateComplete;
+					expect(statusIndicator.getAttribute('state')).to.equal('alert');
+				});
+
+				it('should have state "none" when specified', async() => {
+					statusIndicator.state = 'none';
+					await statusIndicator.updateComplete;
+					expect(statusIndicator.getAttribute('state')).to.equal('none');
+				});
+
+				it('should be subtle', () => {
+					expect(statusIndicator.hasAttribute('bold')).to.be.false;
+				});
+
+			});
+

```

Comment:
I know most of these tests are holdovers from the old components... except for the aXe ones, I question the value of the others now that we have visual diff tests.